LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-decoration-thickness-calc.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>text-decoration-thickness calc() support</title>
+<link rel="author" title="ChangSeok Oh" href="mailto:changseok@webkit.org" />
+<link rel="help" href="https://www.w3.org/TR/css-text-decor-4/#text-decoration-thickness-property" />
+<link rel="match" href="text-decoration-thickness-calc-ref.html" />
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+<meta name="assert" content="Test checks whether text-decoration-thickness supports calc() values.">
+<style type="text/css">
+div {
+  display: flex;
+  font-family: Ahem;
+  font-size: 8px;
```

Comment:
It does seem like the way WebKit caps the text-decoration-thickness is proportional to the font-size. Can we use `font-size: 16px;` and adjust the `em` / `%` accordingly?

Reply:
(sorry for the last minute comment, I noticed that as I was playing around with the testcase)

Reply:
Ah nevermind, your testcase seems to paint fine in WebKit, I got confused with my own JSFiddle.